README.md

```diff
@@ -8,4 +8,14 @@ An example of crate usage can be found in [`src/main.rs`](src/main.rs)
 Requirements
 ------------
 
-Building this (and the dependent crates) requires a [nightly compiler](https://github.com/rust-lang-nursery/rustup.rs/blob/master/README.md#working-with-nightly-rust).
+Building this (and the dependent crates) requires Rust v1.30 or later.
+
+Setup
+------------
+1. Install & compile the latest version of [OpenOCD](http://openocd.org/), then `openocd -f interface/stlink-v2-1.cfg -f target/stm32l0.cfg` and open new terminal session for further work. NOTE: Your target `stlink.cfg` file may change depending on what version of OpenOCD you're running. 
+2. `rustup update`
```

Comment:
Condense into a more narrative description of setup. See [here](https://github.com/rust-embedded/cortex-m-quickstart) and [here](https://japaric.github.io/stm32f103xx-hal/stm32f103xx_hal/) for some nicer examples.